Presentation.Web/Tests/01_Parallel/data-processing/tabs/data-processing-registration.main.e2e.spec.ts

```diff
@@ -30,19 +33,18 @@ describe("Data processing registration main detail tests", () => {
     var dropdownYes = "Ja";
 
     var today = GetDateHelper.getTodayAsString();
-
+    
     beforeAll(() => {
-        loginHelper.loginAsLocalAdmin();
+        loginHelper.loginAsGlobalAdmin();
         testFixture.enableLongRunningTest();
-        dpaHelper.checkAndEnableDpaModule();
+        //dpaHelper.checkAndEnableDpaModule();
```

Comment:
green code?.. make sure this is uncommented since we will get problems with other tests which may disable the module